py3status/modules/thunderbird_calendar.py

```diff
@@ -40,7 +40,7 @@ def _error_response(self, text, color):
         response = {
             'cached_until': time() + self.cache_timeout,
             'full_text': text,
-            'color': color
+            'color': 3s_config['color_bad']
```

Comment:
There is a small issue here where i3s_config is not declared yet, that's why I was passing the color as a parameter.

We could change the color parameter to i3s_config and pass that object to _error_response maybe? Would be less verbose than passing 'color_bad' each time.

`def _error_response(self, text, i3s_config):`